apps/web/src/studio/hooks/useBridgeAPI.ts

```diff
@@ -0,0 +1,44 @@
+import { useMemo } from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { buildBridgeHTTPClient } from '../../api/bridge/bridge.api';
+import { useBridgeURL } from './useBridgeURL';
+
+function useBridgeAPI() {
+  const bridgeURL = useBridgeURL();
+
+  return useMemo(() => buildBridgeHTTPClient(bridgeURL), [bridgeURL]);
+}
+
+export const useDiscover = () => {
+  const api = useBridgeAPI();
+
+  return useQuery(['bridge-workflows'], async () => {
+    return api.discover();
+  });
+};
+
+export const useWorkflow = (templateId: string) => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Prefer object params for something that is likely to grow / change‏